C/include/c4ConnectedClient.h

```diff
@@ -11,11 +11,29 @@
 //
 
 #pragma once
+#include "c4Base.h"
+#include "c4DocumentTypes.h"
```

Comment:
These lines look like glitches from a merge. They were moved to C4ConnectedClientTypes.h yesterday.